software/riscv-tools/Makefile

```diff
@@ -124,13 +124,17 @@ rebuild-newlib:
 	make -C $(TOOLCHAIN_REPO)/build-newlib
 	make -C $(TOOLCHAIN_REPO)/build-newlib install
 
-clean:
-	rm -rf riscv-install
+install-with-builds: checkout-all build-all
+install-clean: install-with-builds clean-builds
```

Comment:
This will be problematic if someone tries to run a parallel build (e.g. -j8). It will run both `install-with-builds` and `clean-builds`.

I don't think anyone runs the build in parallel at the moment. Have you tried?

Reply:
This makefile is not intended to run in parallel... this makefile runs the actual riscv toolchain build in parallel though.

But yeah, this is an issue. I'll add a stamp to ensure clean runs strictly after installation! Thanks :)

Reply:
So we run our AMI build in parallel from the top-level Makefile. That makefile calls this makefile. This would be an issue for us (I think)

Reply:
Got it! I'll fix this...